.gitignore

```diff
@@ -3,8 +3,6 @@
 .build_cache
 ### VisualStudioCode template
 .vscode/*
-!.vscode/settings.json

```

Comment:
Is this change needed?

Reply:
vs.code/settings.json is personal to each users workspace.